TROUBLESHOOTING.md

```diff
@@ -1,7 +1,70 @@
-# Performance Caveats
+# Debugging
+
+Note that the information in this section is subject to be removed in future releases of
+the _PyTorch/XLA_ software, since many of them are peculiar to a given internal implementation
+which might change.
+
+Sometimes, bad things happen and a deeper look into the _PyTorch/XLA_ stack is necessary.
+The _PyTorch/XLA_ stack keeps a series of metrics and counters during its execution.
```

Comment:
Repetition: "_PyTorch/XLA_ stack" - could we make it "We keep a series of metrics ..."?